.github/workflows/lighthouse.yml

```
@@ -0,0 +1,166 @@
+name: Run lighthouse CI When Push on PR to Dev Branch
+on: 
+  pull_request:
+    branches:
+      - dev
+    types: [synchronize, opened]
+
+permissions:
+    contents: read
+    pull-requests: write
+
+jobs:
+  lhci:
+    name: Lighthouse CI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: Use Node.js 20.10.0
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.10.0
```

```suggestion
          node-version-file: .node-version
```

https://github.com/Ludo-SMP/ludo-frontend/blob/dev/.node-version 에 이미 `.node-version` 파일이 있기 때문에 위와 같이 `node-version-file` 을 사용하면 `.node-version` 을 변경했을 때 같이 버전이 업데이트되게 됩니다.

---

반영헸습니다 👍